### PR TITLE
Adds django settings environment variable to all celery processes

### PR DESCRIPTION
**Pull request by dkliban**
re #1216
https://pulp.plan.io/issues/1216

### CODE REVIEW

**Comment by dkliban** [comment]
ok test

File: server/test/unit/server/async/test_app.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""
```
Comment (bmbouter) [inline server/test/unit/server/async/test_app.py]: nice module docblock

File: server/test/unit/server/async/test_app.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from pulp.server.constants import PULP_DJANGO_SETTINGS_MODULE
+
+
+class TestCeleryApp(unittest.TestCase):
+    """
+    Assert that the DJANGO_SETTINGS_MODULE environment variable is set.
+    """
+    @mock.patch('pulp.server.initialization.initialize')
+    def test_django_env_variable(self, mock_initialize):
+        from pulp.server.async import app  # noqa
```
Comment (bmbouter) [inline server/test/unit/server/async/test_app.py]: This should be moved up to the module itself. Having it here doesn't really guard against the problem that other test code could have imported a line of code that set the environment variable. If it was already modified having the import statement here versus at the top won't solve that problem. As such, moving it to the top would be good.

**Comment by bmbouter** [comment]
For the same reasons as [this comment](https://github.com/pulp/pulp/pull/2020#discussion_r38424907), the changes made to `server/test/unit/server/async/test_scheduler.py` can probably be reverted. I don't think this is actually asserting much that the test in `server/test/unit/server/async/test_app.py` isn't already.

**Comment by bmbouter** [comment]
I suggested moving the import statement and also reverting the changes in `server/test/unit/server/async/test_scheduler.py`. I don't need to re-review, LGTM.

**Pull request merged**